### DISCUSSION TITLE
cant install with brew

### DISCUSSION THREAD

**Discussion by ehsanpo** [containerd/nerdctl#1441] [Q&A] [answered]
### Description

Was trying to install nerdcli on Macos Monterey 12.6 using the command

`brew install nerdctl`

Got this error massage :
```
❯ brew install nerdctl
nerdctl: Linux is required for this software.
Error: nerdctl: An unsatisfied requirement failed this build.
```

### Describe the results you received and expected

Be able to install on osx.

### What version of nerdctl are you using?

--

**Comment by AkihiroSuda** [comment, marked as answer]
Expected behavior.
Please try Lima, Rancher Desktop, or Colima to run nerdctl on macOS (via a Linux virtual machine)

> 
> ### Does nerdctl run on macOS ?
> 
> Yes, via a Linux virtual machine.
> 
> [Lima](https://github.com/lima-vm/lima) project provides Linux virtual machines for macOS, with built-in integration for nerdctl.
> 
> ```console
> $ brew install lima
> $ limactl start
> $ lima nerdctl run -d --name nginx -p 127.0.0.1:8080:80 nginx:alpine
> ```
> 
> [Rancher Desktop for Mac](https://rancherdesktop.io/) and [colima](https://github.com/abiosoft/colima) also provide custom Lima machines with nerdctl.
> 


https://github.com/containerd/nerdctl/blob/master/docs/faq.md

**Reply by Zordid** [reply]
Why does this not work, though? I install Lima, I can start and stop machines - but when I type this command I simply get

    ❯ lima nerdctl run -it --rm --platform=linux/amd64 ubuntu
    /bin/bash: line 1: nerdctl: command not found

I don't understand this, as you say its a built-in integration. BTW: I cannot simply install nerdctl using brew as this tells me there's an unsatisfied requirement "Linux"... :(

**Reply by AkihiroSuda** [reply]
Try recreating the instance (`limactl rm -f default && limactl start`).

If it still doesn't work, please report an issue in https://github.com/lima-vm/lima/issues with details.

**Reply by Zordid** [reply]
Would this in any way interfere with Colima? I don't really get how these two play together. Is "default" also the virtual machine that Colima uses to host containers?

**Reply by AkihiroSuda** [reply]
No, colima uses a different instance.

**Reply by Zordid** [reply]
Thanks a lot, I finally have my machine running as a different architecture. Now, the last puzzle piece also gives me a headache: I cannot see how to access my host's file system from within the virtual machine... I wanted to simply access the host's files like described here: https://lima-vm.io/docs/examples/

I used the mount type virtiofs and moun-writeable, but I cannot even find the path inside the machine where I can find my Mac's files... :( What did I miss??

**Reply by AkihiroSuda** [reply]
You may have to specify `--mount <DIR>` in `limactl start`.
Further questions about Lima can be asked in https://github.com/lima-vm/lima/discussions
